### PR TITLE
vpc_peerings implemented as nodes

### PR DESCRIPTION
**Pull request by krisek**
This would be the first step to implement Ec2RouteTables (#410).

### CODE REVIEW

**Review by achantavy — COMMENTED** [review]
Hey, this looks good to me, but can you remind me again why we can't have this logic replace the existing vpc_peering.py, or is deleting that for a future PR?

**Comment by krisek** [comment]
Hi, last time I was trying to delete vpc_peering.py it broke testing.

**Comment by krisek** [comment]
Appearantly this time it worked, what should be the next step?

File: cartography/data/indexes.cypher [hunk 1]
```diff
@@ -112,6 +112,7 @@ CREATE INDEX ON :OktaTrustedOrigin(id);
 CREATE INDEX ON :OktaAdministrationRole(id);
 CREATE INDEX ON :Package(id);
 CREATE INDEX ON :Package(name);
+CREATE INDEX ON :PeeringConnection(id);
```
Comment (achantavy) [inline cartography/data/indexes.cypher:115]: Nit: should we distinguish this somehow as an AWSPeeringConnection? Does this concept exist in other clouds?

**Review by achantavy — DISMISSED** [review]
Thanks!

**Review by achantavy — APPROVED** [review]

**Pull request merged**